### PR TITLE
[MWPW-153654] POST api call for promote

### PR DESCRIPTION
**Pull request by robert-bogos**
## Description
This PR adds the POST api call that triggers the Graybox promote.

## Related Issue
Resolves: [MWPW-153654](https://jira.corp.adobe.com/browse/MWPW-153654)

## Testing instructions
This is part of the Garybox promote feature and all testing regarding it will be done using [this ticket.](https://jira.corp.adobe.com/browse/MWPW-153227)

### CODE REVIEW

**Review by SilviuLCF — APPROVED** [review]
No QA needed at this stage 
Testing will be done in the future on (https://jira.corp.adobe.com/browse/MWPW-153227)

File: libs/blocks/graybox-promote/components/graybox-promote-lit.js [hunk 1]
```diff
@@ -219,7 +219,23 @@ class GrayboxPromote extends LitElement {
         try {
           const { root, gbRoot, driveId } = spData;
           const mainRepo = repo.replace('-graybox', '');
-          const promote = await fetch(`${promoteUrl}?spToken=${this.spToken}&projectExcelPath=${getProjectExcelPath(url)}&rootFolder=${root}&gbRootFolder=${gbRoot}&experienceName=${experienceName}&adminPageUri=${`https://milo.adobe.com/tools/graybox?ref=${ref}%26repo=${mainRepo}%26owner=${owner}`}%26host=business.adobe.com%26project=${mainRepo.toUpperCase()}%26referrer=MOCK_REF&draftsOnly=${promoteDraftsOnly}&promoteIgnorePaths=${promoteIgnorePaths}&driveId=${driveId}&ignoreUserCheck=true`);
+          const params = {
+            adminPageUri: `https://milo.adobe.com/tools/graybox?ref=${ref}&repo=${mainRepo}&owner=${owner}&host=business.adobe.com&project=${mainRepo.toUpperCase()}&referrer=MOCK_REF`,
```
Comment (narcis-radu) [inline libs/blocks/graybox-promote/components/graybox-promote-lit.js:223]: might need to change the `adminPageUri`

**Review by narcis-radu — APPROVED** [review]

File: libs/blocks/graybox-promote/components/graybox-promote-lit.js (hunk 1 above)
Comment (robert-bogos) [reply libs/blocks/graybox-promote/components/graybox-promote-lit.js]: True. Should point to `https://milo.adobe.com/tools/graybox-promote?`. Made the change 👍

**Review by overmyheadandbody — APPROVED** [review]

File: libs/blocks/graybox-promote/components/graybox-promote-lit.js [hunk 2]
```diff
@@ -219,7 +219,23 @@ class GrayboxPromote extends LitElement {
         try {
           const { root, gbRoot, driveId } = spData;
           const mainRepo = repo.replace('-graybox', '');
-          const promote = await fetch(`${promoteUrl}?spToken=${this.spToken}&projectExcelPath=${getProjectExcelPath(url)}&rootFolder=${root}&gbRootFolder=${gbRoot}&experienceName=${experienceName}&adminPageUri=${`https://milo.adobe.com/tools/graybox?ref=${ref}%26repo=${mainRepo}%26owner=${owner}`}%26host=business.adobe.com%26project=${mainRepo.toUpperCase()}%26referrer=MOCK_REF&draftsOnly=${promoteDraftsOnly}&promoteIgnorePaths=${promoteIgnorePaths}&driveId=${driveId}&ignoreUserCheck=true`);
+          const params = {
+            adminPageUri: `https://milo.adobe.com/tools/graybox-promote?ref=${ref}&repo=${mainRepo}&owner=${owner}&host=business.adobe.com&project=${mainRepo.toUpperCase()}&referrer=MOCK_REF`,
```
[the remark this answers is not in the document]
Comment (robert-bogos) [reply libs/blocks/graybox-promote/components/graybox-promote-lit.js:223]: In the long run we won't have any Milo consumer logic inside Milo. The current `business.adobe.com` situation is temporary as we're focused just on BACOM for the moment.

**Pull request merged**